src/Ledger/Certs.lagda

```diff
@@ -37,49 +37,82 @@ cwitness (ccreghot c _)      = c
 \end{figure*}
 
 \begin{figure*}[h!]
+\begin{AgdaAlign}
+\begin{AgdaSuppressSpace}
 \begin{code}
 record CertEnv : Set where
+\end{code}
+\begin{code}[hide]
   constructor ⟦_,_,_,_⟧ᶜ
```

Comment:
I see the rationale for hiding the constructors here and below, since they will be replaced by vertical vectors soon, but then, for consistency, I suppose we should also hide the other such constructors; e.g., those for `UTxOState`, `GovEnv`.

Reply:
Well, that's not a minor request. I made a new ticket for it, #441.